src/test/harness.ts

```diff
@@ -39,7 +38,7 @@ export default async function initialize(options?: TaskClientOptions) {
     account,
     database,
     collection,
-    key,
+    { key },
```

Comment:
how do you intend to test the `ChainedTokenCredential` flow since none of these tests will exercise it?

Reply:
I added a new method of testing this locally. Do you need a solution to test AAD auth as a part of your CI CD or are you publishing this locally?

Reply:
manual testing seems fine for now. i don't think there's a good way to test alternative auth mechanisms with the cosmos emulator so i don't think there's a reasonable way to automate the testing at this time